eng/Versions.props

```diff
@@ -21,13 +21,17 @@
          Can be removed after Arcade moves up. -->
     <MicrosoftVSSDKBuildToolsVersion>16.7.13</MicrosoftVSSDKBuildToolsVersion>
   </PropertyGroup>
+  <!-- Production Dependencies -->
+  <PropertyGroup>
+    <MicrosoftExtensionsDependencyModelVersion>6.0.0-preview.2.21154.6</MicrosoftExtensionsDependencyModelVersion>
+  </PropertyGroup>
   <!-- Toolset Dependencies -->
   <PropertyGroup>
     <!-- DotNetCliVersion MUST match the dotnet version in global.json.
          Otherwise, this version of dotnet will not be installed and the build will error out. -->
-    <DotNetCliVersion>5.0.102</DotNetCliVersion>
+    <DotNetCliVersion>6.0.100-preview.2.21155.3</DotNetCliVersion>
     <MicrosoftNetCompilersToolsetVersion>3.9.0-2.20574.26</MicrosoftNetCompilersToolsetVersion>
-    <NuGetBuildTasksVersion>5.9.1-rc.8</NuGetBuildTasksVersion>
+    <NuGetBuildTasksVersion>5.10.0-preview.1.7148</NuGetBuildTasksVersion>
```

Comment:
Shouldn't darc have been doing this for us? Are we missing a subscription?

Reply:
You're right: https://github.com/dotnet/msbuild/pull/6327 looks to be the last time this version updated automatically. 

I'm thinking it was either the master->main rename or the switch from nuget.client.trusted to nuget.client that broke this.

Reply:
Just going to log my thought process here.

Likely not due to the switch from nuget.client.trusted since we got a few PR's since then. It's more likely that it's darc.

Reply:
Looks like there's no sub for it.
`darc get-subscriptions --channel "VS 16.10"`
```
https://github.com/dotnet/fsharp (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('main')
  - Id: 1b3ed604-a240-43c5-f3e6-08d8e97c775d
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: False
  - Merge Policies: []
  - Last Build: 20210427.5 (b6a352ab0a11a0584348dd77811af7eecaf9d072)
https://github.com/dotnet/fsharp (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('release/5.0.3xx')
  - Id: 8d76063a-5130-4369-8278-08d8e9754760
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: False
  - Merge Policies:
    Standard
  - Last Build: 20210430.7 (e411df7595bbe379f07dd2bcbb6e4046eca9eb31)
https://github.com/dotnet/fsharp (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('release/6.0.1xx-preview4')
  - Id: fec82357-f505-450c-c78c-08d8ff796cf5
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: False
  - Merge Policies:
    Standard
  - Last Build: 20210430.7 (e411df7595bbe379f07dd2bcbb6e4046eca9eb31)
https://github.com/dotnet/msbuild (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('main')
  - Id: 9b9c2fc1-dee5-447e-827b-08d8e9754760
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: False
  - Merge Policies:
    Standard
  - Last Build: 20210428.2 (5b9216a75e98e19eba84e04a5f30bd35a68f317a)
https://github.com/dotnet/msbuild (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('release/5.0.3xx')
  - Id: 08d8569c-a921-4400-1fc7-08d8c8766580
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: False
  - Merge Policies:
    Standard
  - Last Build: 20210428.2 (5b9216a75e98e19eba84e04a5f30bd35a68f317a)
https://github.com/dotnet/msbuild (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('release/6.0.1xx-preview4')
  - Id: f3da650c-3ec0-4a87-5a84-08d8ff79a3bc
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: False
  - Merge Policies:
    Standard
  - Last Build: 20210503.2 (fa96a2a81e0fb8c028057fa204bbf386bfb36aec)
https://github.com/dotnet/roslyn (VS 16.10) ==> 'https://github.com/dotnet/arcade' ('main')
  - Id: 71f51b42-1554-444f-7918-08d8e262a02c
  - Update Frequency: EveryWeek
  - Enabled: True
  - Batchable: True
  - Merge Policies:
    Standard
  - Last Build: 20210501.8 (4c32f5e4e9c0828a94fd4d1c9c0994082c85aaf3)
https://github.com/microsoft/vstest (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('release/5.0.3xx')
  - Id: b4754456-bab5-48e6-093c-08d8ff7930ba
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: False
  - Merge Policies: []
  - Last Build: 20210429-01 (1050e72d7a92a5fb5737aa6a464e8e43e7dc52f4)
https://github.com/nuget/nuget.client (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('main')
  - Id: fcb199f6-ff33-44a0-f3ef-08d8e97c775d
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: False
  - Merge Policies: []
  - Last Build: 5.10.0.7228 (d2bb0de35242b04802c1a7856ebb7718f674565c)
https://github.com/nuget/nuget.client (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('release/5.0.3xx')
  - Id: 61b07feb-68a2-4b3c-64a4-08d8e9757c68
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: False
  - Merge Policies: []
  - Last Build: 5.10.0.7228 (d2bb0de35242b04802c1a7856ebb7718f674565c)
https://github.com/nuget/nuget.client (VS 16.10) ==> 'https://github.com/dotnet/sdk' ('release/6.0.1xx-preview4')
  - Id: 5184e9dc-a7d8-43a0-c78a-08d8ff796cf5
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: False
  - Merge Policies:
    Standard
  - Last Build: 5.10.0.7240 (dca1d060f38e1e02f6bfca41e25f081f19fd534b)
```

Reply:
Correctomundo! `darc add-subscription --channel "VS 16.10" --source-repo "https://github.com/nuget/nuget.client" --target-repo "https://github.com/dotnet/msbuild" --target-branch main --update-frequency "everyWeek" --trigger`

See https://github.com/dotnet/msbuild/pull/6405